controllers/userStatus.js

```diff
@@ -56,6 +56,17 @@ const getUserStatus = async (req, res) => {
   }
 };
 
+/**
+ * Returns sorted user status on the basis of how long they have had that status for and updates user's from field
+ *
+ * @param req {Object} - All user object
+ */
+
+const getSortedUsers = (allUsers) => {
+  const sorted = allUsers.sort((a, b) => a.currentStatus.from - b.currentStatus.from);
```

Comment:
Do we need this manual sort here? Can we use the inbuilt firebase methods to achieve the same?